ha/test/integration/event_analyzer/test_alert_filter.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import json
+import os
+import sys
+import pathlib
+
+# Test case for alert filter
+if __name__ == '__main__':
+    sys.path.append(os.path.join(os.path.dirname(pathlib.Path(__file__)), '..', '..', '..', '..'))
```

Comment:
To be honest, i'm confused - why this line is needed? 
If we need imports from `ha` package in our dev environment - editable installation can do this for us. 
This comment is not an issue to fix, but a question to understand.

Reply:
Currently we not have test framework which will get bundled to rpm.

So to test integration 
what i follow is to setup and provision ha on system then clone ha repo
and execute test

where this line set ha path internally
But this can be removed by setting python path for ha path in env

Also we had same lines in main code also but which was removed
as they are going to bundled in rpm and test tested their

Only test related line not removed as they not currently bundled in rpm

It is to only for quick test but not any specific reason

Reply:
I took reference to other tests and followed the same way. Yes, We can remove this line from all the tests when we have a proper test framework available.

Reply:
Ok, thanks for comments, guys.